stdlib/printexc.ml

```diff
@@ -22,7 +22,9 @@ open Printf
 
 type t = exn = ..
 
-let printers = Atomic.Safe.make []
+type printers : immutable_data = (exn -> string option) Modes.Immutable.t list
```

Comment:
Here the argument is immutable but you're using the modality to enforce the _data part of Immutable_data.